Venter/validate.py

```diff
@@ -0,0 +1,25 @@
+from Venter.models import Header
+
+def csv_file_header_validation(uploaded_csv_file, request):
+    # extracting and converting the header from bytes to string format
```

Comment:
@simranmadhok Please use a docstring for the function specification.

Reply:
Yes, I am currently working on the documentation for the same on a separate 'documentation' branch

Reply:
@simranmadhok Unfollowing the documentation branch practice from now on, yes?

Reply:
Yes